attachments/settings.py

```diff
@@ -0,0 +1,4 @@
+import sys
+import os
+
+ATTACHMENTS_CLAMD = False
```

Comment:
What is this file? Should this be in testapp/settings?

Reply:
Removed in my branch (ATTACHMENTS_CLAMD defaults to false)